react/components/SliderTrack.tsx

```diff
@@ -14,6 +20,16 @@ const isSlideVisible = (
   return index >= currentSlide && index < currentSlide + slidesToShow
 }
 
+const getFirstOrLastVisible = (slidesPerPage: number, index: number) => {
+  if (index % slidesPerPage === 0) {
+    return 'firstVisible'
+  }
+  if ((index + 1) % slidesPerPage === 0) {
+    return 'lastVisible'
+  }
+  return ''
```

Comment:
⛏️ [Nitpick]
```suggestion
  if (index % slidesPerPage === 0) {
    return 'firstVisible'
  }
  
  if ((index + 1) % slidesPerPage === 0) {
    return 'lastVisible'
  }
  
  return ''
```

Reply:
Mind adding a comment to this function, just to explain the conditionals inside those `if`s?
 :)